cypress/e2e/detail_page_spec.cy.ts

```diff
@@ -0,0 +1,31 @@
+describe("BreadDetail Page", () => {
+  beforeEach(() => {
+    cy.intercept(
+      'GET',
+      'https://rails-bread-face5cd9a02c.herokuapp.com/api/v1/breads/Brazil/Pão%20de%20Queijo',
+      {
+        statusCode: 200,
+        fixture: "/mock-country.json",
+      }
+    );
+    cy.visit(encodeURI('/breads/Brazil/Pão de Queijo'));
+  });
+
+  it("should display the bread name and description", () => {
+    cy.get('h2').contains('Pão de Queijo');
```

Comment:
Nice, I feel a lot more confident about looking at this testing after getting walked through cypress with Bobby. Very similar to capybara testing in BE.